dict_DesignatedSurvivor

```diff
@@ -0,0 +1,40 @@
+Tom Kirkman
+Seth Wright 
+Hannah Wells 
+Alex Kirkman 
+ Aaron Shore 
+ Emily Rhodes 
+ Mike Ritter 
+ John Forstell 
+ Jason Atwood 
+ Leo Kirkman
+ Chuck Russink 
+ Kimble Hookstraten 
+ Penny Kirkman 
+Lyor Boone 
+Damian Rennett 
+ Peter MacLeish 
+ Kendra Daynes 
+ Catalan 
+ Wyatt 
+ Ava 
+ Beth MacLeish 
+ Patrick Lloyd 
+ Cornelius Moss 
+ Jay Whitaker 
+ Natalie 
+ Harris Cochrane 
+ Abe Leonard 
+ Jack Bowman 
+ Majid Nassar 
+ Eva Booker 
+Charlie Langdon 
+ Lisa Jordan 
+ Nadia Espinosa 
+ Tiffany Gimble 
+ Dax Minter 
+ Admiral Chernow 
+ Peyton Lane 
+ Detective Blakey
```

Comment:
Please sort the words alphabetically. It's a dictionary file after all and the algorithm relies on it. Applicable on all files you added.